hyperspy/axes.py

```diff
@@ -741,102 +805,18 @@ def _is_increasing_order(self):
         otherwise.
 
         """
-        steps = self.axis[1:] - self.axis[:-1]
-        if np.all(steps > 0):
-            return True
-        elif np.all(steps < 0):
-            return False
-        else:
-            # the axis is not ordered
+        try:
+            steps = self.axis[1:] - self.axis[:-1]
+            if np.all(steps > 0):
+                return True
+            elif np.all(steps < 0):
+                return False
+            else:
+                # the axis is not ordered
+                return None
+        except:
```

Comment:
Specify exception type? Add a comment too?

Reply:
Sounds good! This fails when you have an axis list like ["a","b","c", "d"] so some other value which cannot be subtracted